src/js/gi/components/profile/CalculatorForm.jsx

```diff
@@ -364,7 +364,10 @@ class CalculatorForm extends React.Component {
     return (
       <div>
         <Dropdown
-            label="Will be working"
+            label={this.renderLearnMoreLabel({
+              text: 'Will be working',
+              modal: 'calcWorking'
```

Comment:
Each label that has a "learn more" link uses this function to create a label with an `onClick` handler that is an action to set a specific modal to be visible. This sets the active modal by a key (`calcWorking` here), and the corresponding modal content is in the `Modals.jsx` file.